pkg/codegen/codegen.go

```diff
@@ -39,6 +39,21 @@ type Options struct {
 	SkipFmt            bool // Whether to skip go fmt on the generated code
 }
 
+// imports contains list of imported libraries as alias/fullname tuples, and implements sort.Interface interface
```

Comment:
Would you mind making an `Import` struct, with Alias and Name strings instead of [2]string? It could work exactly the same, but IMO, named objects are much nicer than positional tuples.